client/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.env
```

Comment:
Please add more files/folders to be ignored by git like public, build, IDE related files, etc.
Looks like you have already pushed the public folder. Please remove it.